swcc/swcc/models/project.py

```diff
@@ -362,6 +362,7 @@ def create(self) -> Project:
 
     def download(self, folder: Union[Path, str]):
         session = current_session()
+        self.file.download(folder)
```

Comment:
What is the value of adding this here?  I assume it is needed because of the change in the core models.  I'm not following what the implication is.

Reply:
The implementation of the FileType field we use for `project.file` needs this download method called to convert from a remote reference to a local reference. Without this call, `project.file.path` is None.